VTEX - Checkout API.json

```diff
@@ -9039,11 +9039,118 @@
+                                                        "description": "App ID."
+                                                    },
+                                                    "fields": {
+                                                        "type": "array",
+                                                        "description": "Array of fields available to the app.",
+                                                        "items": {
+                                                            "type": "string"
+                                                        }
+                                                    },
+                                                    "major": {
+                                                        "type": "integer",
+                                                        "description": "App major version."
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "allowMultipleDeliveries": {
+                                            "type": "boolean",
+                                            "description": "On the same purchase, allows the selection of items from multiple delivery channels.",
+                                            "nullable": true
+                                        },
+                                        "allowManualPrice": {
+                                            "type": "boolean",
+                                            "description": "Allows the editing of SKU prices right in the cart.",
+                                            "nullable": true
+                                        },
+                                        "savePersonalDataAsOptIn": {
+                                            "type": "boolean",
+                                            "description": "Allows users to select whether they want the store to keep their personal and payment data saved.",
+                                            "nullable": true
+                                        },
+                                        "maxNumberOfWhiteLabelSellers": {
+                                            "type": "integer",
+                                            "description": "Allows the input of a limit of white label sellers involved on the cart.",
+                                            "nullable": true
+                                        },
+                                        "maskFirstPurchaseData": {
+                                            "type": "boolean",
+                                            "description": "Allows, on a first purchase, masking client's data. It could be useful when a shared cart is used and the client doesn't want to share its data.",
```

Comment:
```suggestion
                                            "description": "Allows, on a first purchase, masking client's data. It could be useful when a shared cart is used and the client does not want to share its data.",
```